### PR TITLE
blog: Add Ops friendly APISIX post

### PR DESCRIPTION
**Pull request by nfrankel**
Screenshot 2023-08-10 at 10 53 32

### CODE REVIEW

**Comment by pottekkat** [comment]
**Don't merge until August 17th**

**Review by yzeng25 — DISMISSED** [review]

**Review by pottekkat — DISMISSED** [review]

**Comment by pottekkat** [comment]
We need to update the cover image.

File: blog/en/blog/2023/08/17/ops-friendly-apisix.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+  - Operations
+  - Ops
+  - DevOps
+  - Configuration
+description: >
+  Though I always worked on the Dev side of IT, I was also interested in the Ops side. I even had a short experience being a WebSphere admin: I used it several times, helping Ops deal with the Admin console while being a developer.
+  Providing a single package that Ops can configure and deploy in different environments is very important. As a JVM developer, I've been happy using Spring Boot and its wealth of configuration options: command-line parameters, JVM parameters, files, profiles, environment variables, etc.
+  In this short post, I'd like to describe how you can do the same with Apache APISIX in the context of containers.
+tags: [Ecosystem]
+image: https://static.apiseven.com/uploads/2023/08/10/0Lvnto4O_honey-bees-337695.jpg
```
Comment (Yilialinn) [inline blog/en/blog/2023/08/17/ops-friendly-apisix.md]:
```suggestion
image: https://static.apiseven.com/2022/10/19/634f6677742a1.png
```

Comment (pottekkat) [reply blog/en/blog/2023/08/17/ops-friendly-apisix.md]: Thank you @Yilialinn 🤝

**Review by pottekkat — APPROVED** [review]

**Comment by pottekkat** [comment]
Ready to merge.

**Review by yzeng25 — APPROVED** [review]

**Pull request merged**